std/process.d

```diff
@@ -1443,6 +1499,33 @@ int wait(Pid pid) @safe
     else version (Posix) assert(pid.osHandle < 0);
 }
 
+@system unittest // Pid.fromSystemPid()
+{
+    version (Windows)    TestScript prog = "do { timeout 1 } while (1)";
```

Comment:
> Well, it's not being executed as a PowerShell script. Look at the implementation of TestScript. The string is saved to a .cmd file. [..]

I have missed that. I just assumed that if the syntax was invalid, auto tester would complain about it.

> In addition to fixing the script, you need to find why your test was broken (in that it allowed a broken script to work and pass the test).

Again, I just assumed everything was ok since the tester did not complain about it.
Any suggestions on how to approach the issue and find out why the tester is not complaining?